src/frontend/ReportPoints.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace tuja\frontend;
+
+
+use Exception;
+use tuja\data\model\Group;
+use tuja\data\model\Station;
+use tuja\data\store\StationDao;
+use tuja\data\store\StationPointsDao;
+use tuja\frontend\AbstractCrewMemberView;
+use tuja\frontend\router\ReportPointsInitiator;
+use tuja\util\concurrency\LockValuesList;
+use tuja\view\FieldNumber;
+
+class ReportPoints extends AbstractCrewMemberView {
```

Comment:
- [ ] See if there is more shared code which can be move to AbstractCrewMemberView